go/oasis-test-runner/cmd/cmp.go

```diff
@@ -0,0 +1,517 @@
+package cmd
```

Comment:
Consider moving this to its own directory/package, e.g. `cmd/cmp`.

Reply:
Done. To avoid circular imports I had to add another `cmd/common` package containing constants and shared variables.